tests/test_normalize_by_median.py

```diff
@@ -581,7 +580,8 @@ def test_diginorm_basic_functionality_4():
                     infile)
 
     script = 'normalize-by-median.py'
-    args = list(CUTOFF) + list(PAIRING) + ['-k', '15', infile]
+    args = list(CUTOFF) + ['-k', '15', infile]
```

Comment:
requiring pairing error'd out--there's unpaired reads in the input (which makes sense since -p used to be used to include single sets of paired/unpaired stuff and this test was never actually running, so it didn't break when we made the change within NBM)